### PR TITLE
Fix: Removed fetch-fonts from the build scripts so travis stops erroring

### PR DESCRIPTION
**Pull request by colincclark**
Removing the fetch-fonts step from the publish storybook step so travis stops failing at this point. As this occurs at the deploy stage, we can only test this once merged I think

### CODE REVIEW

**Comment by times-tools** [comment]
Please find visual snapshots of your changed components here: https://www.bitrise.io/build/a883d3d37907e37e

File: package.json [hunk 1]
```diff
@@ -27,7 +27,7 @@
     "fetch-fonts": "node ./lib/fetch-fonts",
     "prestorybook": "yarn fetch-fonts && yarn storybook:build-vendor",
     "storybook": "start-storybook -p 9001 -c .storybook -s ./dist/public",
-    "prestorybook:build": "yarn prestorybook",
+    "prestorybook:build": "yarn storybook:build-vendor",
```
Comment (craigbilner) [inline package.json]: I think this has got quite confusing now - only the static storybook script uses this so doesn't need to be a pre script?

Comment (colincclark) [reply package.json]: I find all the pre scripts quite confusing. The other option here is:

`"storybook:build": "yarn storybook:build-vendor && build-storybook -c .storybook -s ./dist/public",`

is that what you had in mind @craigbilner ?

Comment (craigbilner) [reply package.json]: yah

**Comment by times-tools** [comment]
Please find visual snapshots of your changed components here: https://www.bitrise.io/build/62192d268eac9c0a

**Comment by colincclark** [comment]
@craigbilner changes have been implemented

File: package.json (hunk 1 above)
Comment (colincclark) [reply package.json]: This change was implemented

**Pull request merged**